src/shared/ui/LongTextInput.tsx

```diff
@@ -0,0 +1,52 @@
+import { TextInput } from 'react-native';
+
+import { GetProps, setupReactNative, styled } from '@tamagui/core';
+import { focusableInputHOC } from '@tamagui/focusable';
+
+setupReactNative({
+  TextInput,
+});
+
+export const LongTextInput = styled(
+  TextInput,
+  {
+    name: 'ParagraphText',
+
+    m: 0,
+    px: 12,
+    py: 8,
+
+    minWidth: 0,
+    minHeight: 56,
+    maxHeight: 350,
+    maxWidth: '100%',
+    width: '100%',
```

Comment:
`minWidth`, `maxWidth` and `width` are probably not all required, just `width`

Reply:
I Removed